bundles/CoreBundle/Resources/translations/en.json

```diff
@@ -167,6 +167,7 @@
     "locked": "Locked",
     "element_cannot_be_move_because_it_is_locked": "Element cannot be moved because it is locked.",
     "element_cannot_be_moved": "The element cannot be moved here",
+    "element_cannot_be_moved_because_target_is_paginated": "The element cannot be moved here because the target is a paginated tree/folder",
```

Comment:
we should also suggest to use "Search & Move" option under Advance Menu for still required cases.